testing/opae-c/test_error_c.cpp

```diff
@@ -105,10 +108,22 @@ TEST_P(error_c_p, read) {
  *             and the fn returns FPGA_OK.<br>
  */
 TEST_P(error_c_p, get_info) {
-  fpga_error_info info;
-  EXPECT_EQ(fpgaGetErrorInfo(tokens_[0], 0, &info), FPGA_OK);
-  EXPECT_STREQ(info.name, "first_error");
-  EXPECT_EQ(info.can_clear, false);
+  fpga_properties props;
+  uint32_t num_errors = 0;
+  ASSERT_EQ(fpgaGetProperties(tokens_[0], &props), FPGA_OK);
+  ASSERT_EQ(fpgaPropertiesGetNumErrors(props, &num_errors), FPGA_OK);
+  // this is a port, which only has three error registers
```

Comment:
Should we make such assumptions?  ie, will this be platform agnostic?

Reply:
right now, yes (for ports)